tests/cases/compiler/strictFunctionTypes1.ts

```diff
@@ -0,0 +1,18 @@
+// @strict: true

```

Comment:
Can you add `@declare: true` so that we get a baseline for d.ts output?

Reply:
+1